apps/namadillo/src/atoms/settings/services.ts

```diff
@@ -2,7 +2,7 @@ import { DefaultApi } from "@anomaorg/namada-indexer-client";
 import toml from "toml";
 import { SettingsTomlOptions } from "types";
 
-export const isIndexerAlive = async (url: string): Promise<boolean> => {
+export const isApiAlive = async (url: string): Promise<boolean> => {
```

Comment:
I understand what you've done here :) and I think it's a very clean idea, however I'm concern that it might not be future-proof. The RPC and the indexer have two completely different APIs and purposes, although the /health endpoint coincides. However, DefaultApi it's an external library that only queries things in the indexer, and it's susceptible to changes. So I think, for this specific case, we should add some replication to ensure everything will be always fine

Reply:
Yeah, makes sense, I'll improve it 👍

Reply:
We need to make a decision here as don't have a query library on namadillo yet. I can install `axios` to resolve this, what to do you think?

Another alternative is to create a new instance of the sdk and add a method there to check the health endpoint, but I believe it's a bazooka for a simple task

Reply:
done!